packages/client/src/utils/errors.tsx

```diff
@@ -8,6 +8,23 @@ import { tryFindMessageDescriptor } from "@gc-digital-talent/i18n";
 export const extractErrorMessages = (combinedError: CombinedError) =>
   combinedError.graphQLErrors.flatMap((error) => error.message);
 
+// grab the validation error enums to map them to messages, guarding against possible missing fields
+export const extractValidationMessageKeys = (
+  combinedError: CombinedError,
+): string[] | null => {
+  const { extensions } = combinedError.graphQLErrors[0];
+  if (extensions && extensions.validation) {
+    const validationObject = extensions.validation as object;
```

Comment:
Using `as` to override the type checker makes me nervous.  I'd prefer to see a real check like `typeof extensions.validation === 'object'`.